lib/dgram.js

```diff
@@ -350,19 +347,6 @@ Socket.prototype.bind = function(port_, address_ /* , callback */) {
   return this;
 };
 
-
-function validatePort(port) {
-  const legal = isLegalPort(port);
-  if (legal)
-    port = port | 0;
```

Comment:
This seems to change the behavior? Zero was prohibited before.

Reply:
Ugh. ok

Reply:
Ok, not allowing zero is now accounted for.